shell/platform/linux/fl_text_input_plugin.cc

```diff
@@ -47,6 +47,8 @@ static constexpr char kTextAffinityDownstream[] = "TextAffinity.downstream";
 static constexpr char kMultilineInputType[] = "TextInputType.multiline";
 static constexpr char kNoneInputType[] = "TextInputType.none";
 
+static constexpr char kFlTextInputActionSend[] = "TextInputAction.send";
```

Comment:
Nit: Maybe `kSendInputType` in order to align with the other input type constants above?